test/v2/load/load_generator_metrics.go

```diff
@@ -21,20 +26,105 @@ func newLoadGeneratorMetrics(registry *prometheus.Registry) *loadGeneratorMetric
 			Name:      "operations_in_flight",
 			Help:      "Number of operations in flight",
 		},
+		[]string{"operation"},
+	)
+
+	writeSuccesses := promauto.With(registry).NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "write_successes",
+			Help:      "Number of write successes",
```

Comment:
nit: For the write metrics the help documentation is not clear where we are writing.